src/aspire/operators/blk_diag_matrix.py

```diff
@@ -679,6 +680,21 @@ def check_psd_matrix(self):
         )
         return np.alltrue(eigenvalues > 0.0)
 
+    def make_psd_matrix(self):
```

Comment:
Just `make_psd` is enough.

Reply:
Changed to `make_psd` in new commit 8d08e36.